arm-apimanagement/2015-09-15/swagger/service.json

```diff
@@ -0,0 +1,1288 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "APIManagementServiceClient",
+    "description": "REST API for Azure APIManagement Service",
+    "version": "2015-09-15"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_GetApiManagementServiceOperation",
```

Comment:
We follow the Noun_Verb pattern for OperationId. It would be nice if the operationId is like `"ApiManagement_GetServiceOperation"` minus the repeating noun.
this translates to client.ApiManagement.GetServiceOperation() and also looks better in intellisense.

@tbombach - We must have this regex check in the Linter in a future release :).